lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.CoilHeatingDXSingleSpeed.rb

```diff
@@ -0,0 +1,149 @@
+class ASHRAE901PRM < Standard
+  # @!group CoilHeatingDXSingleSpeed
+
+  include ASHRAEPRMCoilDX
+
+  # Finds capacity in W.  This is the cooling capacity of the
+  # paired DX cooling coil.
+  #
+  # @return [Double] capacity in W to be used for find object
```

Comment:
The methods in this file are missing YARD comments for the parameters.

Reply:
Added